src/utils/array.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const chunkArray = (arr: any, size: number) => {
```

Comment:
what's `chunkArray`? I have even seen the usage and is difficult to grasp what it does without reading implementation details. There is any better name we can assign? Or perhaps my English is failing?

Reply:
Chunk is like splitting, I found that is actually a pretty common approach for this need. What this function allow us is to do the following:

```
const myArray = [1,2,3,4];

const chunkedArray = chunkArray(myArray, 2);
console.log(chunkedArray) // [ [1, 2], [3, 4] ]
```

Reply:
What about using lodash for utilities like this, https://lodash.com/docs/#chunk, paired with https://github.com/lodash/lodash-webpack-plugin shouldn't affect the payload size too much